client/src/test/java/com/sinch/sdk/domains/verification/adapters/VerificationsServiceTest.java

```diff
@@ -51,28 +66,104 @@ public void initMocks() {
   }
 
   @Test
-  void start() throws ApiException {
+  void startSms() throws ApiException {
 
     when(api.startVerification(eq(startVerificationSMSRequestDto)))
         .thenReturn(StartVerificationResponseDtoTest.expectedStartVerificationSMSDto);
 
     StartVerificationResponse response =
-        service.start(VerificationsDtoConverterTest.startVerificationSMSRequest);
+        service.startSms(VerificationsDtoConverterTest.startVerificationSMSRequest);
 
     Assertions.assertThat(response)
         .usingRecursiveComparison()
         .isEqualTo(VerificationsDtoConverterTest.expectedStartVerificationSMSResponse);
   }
 
   @Test
-  void getByIdentity() throws ApiException {
+  void startFlashCall() throws ApiException {
+
+    when(api.startVerification(eq(startVerificationFlashCallRequestDto)))
+        .thenReturn(StartVerificationResponseDtoTest.expectedStartVerificationFlashCallDto);
+
+    StartVerificationResponse response =
+        service.startFlashCall(VerificationsDtoConverterTest.startVerificationFlashCallRequest);
+
+    Assertions.assertThat(response)
+        .usingRecursiveComparison()
+        .isEqualTo(VerificationsDtoConverterTest.expectedStartVerificationFlashCallResponse);
+  }
+
+  @Test
+  void startCallout() throws ApiException {
+
+    when(api.startVerification(eq(startVerificationCalloutRequestDto)))
+        .thenReturn(StartVerificationResponseDtoTest.expectedStartVerificationCalloutDto);
+
+    StartVerificationResponse response =
+        service.startCallout(VerificationsDtoConverterTest.startVerificationCalloutRequest);
+
+    Assertions.assertThat(response)
+        .usingRecursiveComparison()
+        .isEqualTo(VerificationsDtoConverterTest.expectedStartVerificationCalloutResponse);
+  }
+
+  @Test
+  void startSeamless() throws ApiException {
+
+    when(api.startVerification(eq(startVerificationSeamlessRequestDto)))
+        .thenReturn(StartVerificationResponseDtoTest.expectedStartVerificationSeamlessDto);
+
+    StartVerificationResponse response =
+        service.startSeamless(VerificationsDtoConverterTest.startVerificationSeamlessRequest);
+
+    Assertions.assertThat(response)
+        .usingRecursiveComparison()
+        .isEqualTo(VerificationsDtoConverterTest.expectedStartVerificationSeamlessResponse);
+  }
+
+  @Test
+  void reportSmsByIdentity() throws ApiException {
+
+    when(api.reportVerificationByIdentity(
+            eq("number"), eq("endpoint string"), eq(verificationReportSMSRequestDto)))
+        .thenReturn(VerificationReportDtoTest.expectedVerificationCalloutDto);
```

Comment:
Should be a SMS response

Reply:
Of course !
Thank you
Fixed(and added missing proper dedicated type returned by service APIs)